instrumentation/shoryuken/test/opentelemetry/instrumentation/shoryuken/middlewares/server/tracer_middleware_test.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+require_relative '../../../../../../lib/opentelemetry/instrumentation/shoryuken'
+require_relative '../../../../../../lib/opentelemetry/instrumentation/shoryuken/middlewares/server/tracer_middleware'
+
+describe OpenTelemetry::Instrumentation::Shoryuken::Middlewares::Server::TracerMiddleware do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Shoryuken::Instrumentation.instance }
+  let(:exporter) { EXPORTER }
+  let(:spans) { exporter.finished_spans }
+  let(:enqueuer_span) { spans.first }
+  let(:job_span) { spans.last }
+  let(:root_span) { spans.find { |s| s.parent_span_id == OpenTelemetry::Trace::INVALID_SPAN_ID } }
+  let(:config) { {} }
+  let(:queue_name) { 'default' }
+  let(:sqs_msg) do
+    double(
+      Shoryuken::Message,
+      queue_url: queue_name,
+      body: '{"test" : 1}',
+      message_attributes: {
+        'shoryuken_class' => { string_value: worker_class.name }
+      },
+      message_id: SecureRandom.uuid,
+      receipt_handle: SecureRandom.uuid
+    )
+  end
+  let(:sqs_client) { Aws::SQS::Client.new(stub_responses: true) }
+  let(:worker_class) { SimpleJob }
+
+  before do
+    allow(sqs_client).to receive(:get_queue_url).and_return(double(queue_url: 'https://sqs.fake.amazonaws.com/1/queue-name'))
+    allow(sqs_msg).to receive(:[]).with('baggage')
```

Comment:
:thinking: removing this and following line will reveal the error in tests as well